iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergMergeTaskProperties.java

```diff
@@ -42,14 +45,23 @@ public Path getTmpLocation() {
     return new Path(location + "/data/");
   }
 
-  public StorageFormatDescriptor getStorageFormatDescriptor() throws IOException {
-    FileFormat fileFormat = FileFormat.fromString(properties.getProperty(TableProperties.DEFAULT_FILE_FORMAT,
-            TableProperties.DEFAULT_FILE_FORMAT_DEFAULT));
-    StorageFormatDescriptor descriptor = storageFormatFactory.get(fileFormat.name());
-    if (descriptor == null) {
-      throw new IOException("Unsupported storage format descriptor");
+  @Override
+  public Map<Path, Object> getSplitProperties() throws IOException {
```

Comment:
why it's called `getSplitProperties` if it returns data file abstraction (ContentFile)?

Reply:
Well the idea is that this can include anything which is related to the split itself. Hence I have generically added it as a split property.

Reply:
why not to return `Properties` object in that case? I think it's better to use proper types/names and avoid casting latter

Reply:
Done.